app/controllers/challenge_partnerships_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class ChallengePartnershipsController < ApplicationController
+  include Pundit
```

Comment:
Isn't pundit included by default?

Reply:
Not from the application controller, only the base controllers of each site section